content/asset-tracking/using-the-sdks.textile

```diff
@@ -160,19 +160,45 @@ blang[kotlin,swift].
 
   h3(#publisher-initializing). Initializing the Publisher
 
-  During initialization of the publisher various methods can be called to configure the @Publisher@.
+  During initialization of the publisher various methods can be called to configure the @Builder@ interface of the @Publisher@.
 
   The required methods are:
 
   - connection := Called to provide Ably connection information, such as API keys, and any other configuration parameters as needed.
   - map := Called to provide Mapbox configuration, such as API keys, any other configuration parameters as needed.
-  - resolutionPolicy := Sets the policy factory to be used to define the target resolution for publishers created from this builder.
   - androidContext := Called to provide the Android runtime context (on Android only).
+  - resolutionPolicy := Sets the policy factory to be used to define the target resolution for publishers created from this builder.
+  - backgroundTrackingNotificationProvider := Sets the notification that will be displayed for the background tracking service. Please note that this notification will be removed when you call the @stop@ method (on Android only).
+  - start := Creates a @Publisher@ and starts publishing. The returned publisher instance does not start in a state whereby it is actively tracking anything. If tracking is required from the outset then the @Publisher.track@ method must be subsequently called. In order to detect the device's location @ACCESS_COARSE_LOCATION@ or @ACCESS_FINE_LOCATION@ permission must be granted.
```

Comment:
To be super precise, users can call both `@Publisher.track@` and `@Publisher.add@` to start tracking :wink: